frameworks/spark/README.md

```diff
@@ -0,0 +1 @@
+Template README
```

Comment:
Should replace this with something better.  Doesn't have to be perfect, but `Template README` should be replaced.